src/components/inputs/toggle-input/toggle-input.js

```diff
@@ -61,29 +66,29 @@ const Span = styled.span`
 `;
 
 const Input = styled.input`
-  &:checked + ${Span}:before {
+  &:checked + ${Span}::before {
     background: ${vars.colorPrimary};
   }
 
-  &:disabled + ${Span}:before {
+  &:disabled + ${Span}::before {
     background: ${vars.colorNeutral};
   }
 
-  &:disabled&:checked + ${Span}:before {
+  &:disabled&:checked + ${Span}::before {
     background: ${vars.colorPrimary25};
   }
 
-  &:checked + ${Span}:after {
+  &:checked + ${Span}::after {
```

Comment:
should use `::after` and not `:after`

Reply:
Ah nice. I always wondered if there was a preferred way.